OsmAnd/build.gradle

```diff
@@ -39,18 +39,13 @@ android {
 	compileOptions {
 		sourceCompatibility = JavaVersion.VERSION_17
 		targetCompatibility = JavaVersion.VERSION_17
-		coreLibraryDesugaringEnabled true
```

Comment:
Does this plugin have issues with desugaring or kotlinOptions?

Reply:
I'm not aware of such issues, please feel free to adjust.